Header header
string name
int32 id
bool status

float64 b_level

float32 x
float32 y
float32 theta